integration_tests/e2e/courtCaseAlternativeSentenceLength.cy.ts

```diff
@@ -11,7 +11,7 @@ context('Court Case Alternative Sentence Length Page', () => {
     const courtCaseWarrantTypePage = Page.verifyOnPage(CourtCaseWarrantTypePage)
     courtCaseWarrantTypePage.radioLabelSelector('SENTENCING').click()
     courtCaseWarrantTypePage.continueButton().click()
-    cy.visit('/person/A1234AB/add-court-case/0/add-court-appearance/0/alternative-overall-sentence-length')
+    cy.visit('/person/A1234AB/add-court-case/0/add-court-appearance/0/SENTENCING/alternative-overall-sentence-length')
```

Comment:
I moved the SENTENCING specific pages into their own SENTENCING route. I was contemplating whether to pass in :warratnType into the url but decided against it as these routes are only achievable by the SENTENCING route alone. So, used SENTENCING (upper-case) to keep it consistent with the actual warrantType